scss/_wrapped_common.scss

```diff
@@ -29,6 +29,11 @@
 		@include oTypographySub;
 	}
 
+	ol,
+	ul {
+		@include oTypographyBodyList;
```

Comment:
It's a major release, so let's do it. We might need to add deprecation warnings to the current release though, for use cases that are being renamed, and for the ones that are being removed.